gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -666,7 +688,7 @@ private ManagedChannel createSingleChannel() throws IOException {
             channelCredentials =
                 CompositeChannelCredentials.create(channelCredentials, mtlsS2ACallCredentials);
           }
-          builder = Grpc.newChannelBuilder(endpoint, channelCredentials);
+          builder = Grpc.newChannelBuilder(mtlsEndpoint, channelCredentials);
```

Comment:
nit: Small comment that the fallback uses mTLS not non-MTLS because of reason X

Reply:
Done.